controller/app/validators/key_content_validator.rb

```diff
@@ -0,0 +1,15 @@
+class KeyContentValidator < ActiveModel::Validator
+   def validate(record)
+     key = Net::SSH::KeyFactory.load_data_public_key(record.type + " " + record.content)
```

Comment:
@sdodson Sounds good. I'll see what I can do as well. As for the net-ssh version: Yes, 2.7.0 is the one I have been using with the customer and there is no reason for changing it in the channels. However, just as an additional data point, I also had one that used net-ssh 2.9.x and it worked just as well.